src/components/MyKiva/MyKivaContainer.vue

```diff
@@ -21,4 +21,11 @@ import KvGrid from '@kiva/kv-components/vue/KvGrid';
 	max-width: 67rem;
 	@apply tw-mx-auto;
 }
+
+.container > div {
```

Comment:
This is applying to the entire page, what was the issue and does this have any other affects?